manifests/l/LGUG2Z/whkd/0.1.1/LGUG2Z.whkd.locale.en-US.yaml

```
@@ -0,0 +1,35 @@
+# Created with YamlCreate.ps1 v2.2.2 $debug=NVS0.CRLF.7-3-2.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: LGUG2Z.whkd
+PackageVersion: 0.1.1
+PackageLocale: en-US
+Publisher: LGUG2Z
```

```suggestion
Publisher: Jade Iqbal
```
This must match Control Panel.